src/main/java/com/twilio/http/ValidationClient.java

```diff
@@ -36,11 +39,19 @@
      * @param  privateKey Private Key
      */
     public ValidationClient(String accountSid, String credentialSid, String signingKey, PrivateKey privateKey) {
-        RequestConfig config = RequestConfig.custom()
-            .setConnectTimeout(CONNECTION_TIMEOUT)
-            .setSocketTimeout(SOCKET_TIMEOUT)
-            .build();
+        this(accountSid, credentialSid, signingKey, privateKey, DEFAULT_REQUEST_CONFIG);
```

Comment:
Can we please have a test, likely [here](https://github.com/twilio/twilio-java/blob/main/src/test/java/com/twilio/http/TwilioRestClientTest.java)?

Reply:
I added a new unit test: ValidationClientTest.java